### PR TITLE
[1LP][RFR][NOTEST] CB: new manual tests

### PR DESCRIPTION
**Pull request by digitronik**
Signed-off-by: Nikhil Dhandre <ndhandre@redhat.com>

Purpose or Intent
=================
Previously only `VM` object supporting open url and new support going to be add in next release for now adding manual test.

- Added as per custom button test structure.... so as automate ids going to match with manual.

### CODE REVIEW

File: cfme/tests/automate/custom_button/test_generic_objects.py [hunk 1]
```diff
@@ -316,3 +316,27 @@ def test_custom_button_expression_evm_obj(appliance, request, setup_obj, button_
             assert not custom_button_group.is_displayed
             setup_obj.add_tag(tag)
             assert button.text in custom_button_group.items
+
+
+@pytest.mark.manual
+def test_custom_button_open_url_evm_obj(setup_obj, button_group):
```
Comment (john-dupuy) [inline cfme/tests/automate/custom_button/test_generic_objects.py]: Possible to parametrize this test against the different objects? Or is this something you'd like to wait to do until you automate the tests? 

Also could you reference the RFE BZ in here?

Comment (digitronik) [reply cfme/tests/automate/custom_button/test_generic_objects.py]: yup @john-dupuy 
`evm` object need to set as per `object type`. so I need to modify fixture means need a method fixture where i can send object type. I won't like to automate before feature comes in downstream. yup easy to automate for me just waiting.

Comment (digitronik) [reply cfme/tests/automate/custom_button/test_generic_objects.py]: @john-dupuy as you know I like to collect BZ. but this is RFE when just they take decision to add `open url` for other objects. I would not treat it as BZ. Actually `startsin` pointing this. but still if you want let me know.

Comment (digitronik) [reply cfme/tests/automate/custom_button/test_generic_objects.py]: I think better to add let me add this :)

Comment (john-dupuy) [reply cfme/tests/automate/custom_button/test_generic_objects.py]: I'd say it's optional, I've been adding them just as a reference to help understand the test case, but I wouldn't say that it's necessary.

Comment (digitronik) [reply cfme/tests/automate/custom_button/test_generic_objects.py]: yup totally agree with you... better to have `BZ` collection against test. It helps me lot. just i search `BZ` number in `IDE` and I am getting related tests. personally its help me lot. I don't know about other team member. but i found some of us fallowing `Bugzilla` tag.

File: cfme/tests/automate/custom_button/test_generic_objects.py [hunk 2]
```diff
@@ -316,3 +316,30 @@ def test_custom_button_expression_evm_obj(appliance, request, setup_obj, button_
...
+        startsin: 5.11
+        casecomponent: CustomButton
+        tags: custom_button
+        testSteps:
+            1. Create ruby method for url functionality
+            2. Create custom button group with the Object type
+            3. Create a custom button with open_url option and respective method
+            4. Navigate to object Detail page
+            5. Execute custom button
+            6. Check new tab open or not with respective url
```
Comment (jawatts) [inline cfme/tests/automate/custom_button/test_generic_objects.py]: I would expect to see `testSteps` and `expectedResults` for a manual test case here.

**Comment by jawatts** [comment]
waiting for travis

**Pull request merged**